### PR TITLE
Add Boundary Error and Made Troubleshooting Guide

### PR DESCRIPTION
**Pull request by AvinaashGhansam**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,17 +20,19 @@
...
     "@react-spring/three": "^9.7.3",
     "@react-three/drei": "^9.88.16",
-    "@react-three/fiber": "^8.15.11",
     "@react-three/postprocessing": "^2.15.11",
     "framer-motion": "^10.16.5",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-error-boundary": "^4.0.12",
     "react-icons": "^4.12.0",
+    "react-router-dom": "^6.21.3",
```
Comment (zlix12) [inline package.json:35]: This should not be needed any longer right?

File: package.json [hunk 2]
```diff
@@ -20,17 +20,19 @@
     "@ar-js-org/ar.js": "^3.4.5",
     "@ar-js-org/artoolkit5-js": "^0.3.0",
     "@artcom/react-three-arjs": "^0.5.6",
+    "@chakra-ui/icons": "^2.1.1",
     "@chakra-ui/react": "^2.8.2",
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
     "@react-spring/three": "^9.7.3",
     "@react-three/drei": "^9.88.16",
-    "@react-three/fiber": "^8.15.11",
```
Comment (zlix12) [inline package.json:28]: Why did this get removed?

File: src/view/ViewManager.tsx [hunk 3]
```diff
@@ -13,5 +13,4 @@ const ViewManager: React.FC = () => {
 
   return <CurrentViewComponent changeView={setCurrentView} />;
 };
-
```
Comment (zlix12) [inline src/view/ViewManager.tsx:16]: Is this change necessary?

File: src/App.tsx [hunk 4]
```diff
@@ -1,20 +1,21 @@
-import './App.css';
```
Comment (zlix12) [inline src/App.tsx:1]: Why did this get removed?

File: src/view/ErrorFallBack.tsx [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+  Text,
+  UnorderedList,
+  ListItem,
+  Collapse,
+  Box,
+  ChakraProvider
+} from '@chakra-ui/react';
+import { useState } from 'react';
+
+function ErrorFallback() {
```
Comment (zlix12) [inline src/view/ErrorFallBack.tsx:12]: Nit: Can we just call this troubleshooting guide? There's nothing here that necessarily couples it to being an error fallback

File: package.json [hunk 6]
```diff
@@ -20,17 +20,19 @@
     "@ar-js-org/ar.js": "^3.4.5",
     "@ar-js-org/artoolkit5-js": "^0.3.0",
     "@artcom/react-three-arjs": "^0.5.6",
+    "@chakra-ui/icons": "^2.1.1",
```
Comment (zlix12) [inline package.json:23]: Why was this added? I'm not seeing where it is used

File: src/view/ErrorFallBack.tsx [hunk 7]
```diff
@@ -0,0 +1,226 @@
...
+          <Heading size="xl">Something Went Wrong...</Heading>
+          <Heading size="md" mt={4} color="magenta.500">
+            Troubleshooting Guide
+          </Heading>
+          <Text>
+            If you&apos;re experiencing issues, follow the steps below to
+            troubleshoot and resolve common problems:
+          </Text>
+          <UnorderedList>
+            <ListItem>
```
Comment (zlix12) [inline src/view/ErrorFallBack.tsx:32]: Can these list items be abstracted into a reusable component? Seems that some duplicate code could be reduced

**Review by zlix12 — CHANGES_REQUESTED** [review]

File: package.json (hunk 6 above)
Comment (AvinaashGhansam) [reply package.json:23]: I was initially using it but ill remove it

File: package.json (hunk 2 above)
Comment (AvinaashGhansam) [reply package.json:28]: not sure but ill add it back

**Review by zlix12 — APPROVED** [review]
LGTM
